docs/developer/events.rst

```diff
@@ -15,6 +15,40 @@ showInfoPanel Object Show the info panel in the right.
 hideInfoPanel NULL   Hide the info panel displayed in the right. 
 ============= ====== =========================================== 
 
+The `k-info-panel` can be triggered by a `k-button` or a `k-action-menu` or
+anything that calls the method created to set-up a new `k-info-panel`.
+
+**Example**
+
+.. code-block:: html
+
+    <template>
+        ...
+    </template>
+
+    <script>
+    /* All the javascript methods are optional */
+    module.exports = {
+        methods: {
+            showInfoPanel()  {
+                var content = {
+                    "component": 'search-hosts',
+                    "content": {}, /* Content used in the component */
+                    "icon": "search",
+                    "title": "Search Hosts",
+                    "subtitle": "by kytos/UI Test"

```

Comment:
What's the meaning of this subtitle?

Reply:
thats the subtitle for the infoPanel: 
![image](https://user-images.githubusercontent.com/17555995/98691920-4e6aad00-234d-11eb-8d72-1ea278020f76.png)